app/Http/Controllers/Admin/DiagramController.php

```diff
@@ -342,6 +381,21 @@ public function showDesignWithoutDiagram(Request $request, $id)
         return view('test.workspace.design_without_diagram', compact('workspace'));
     }
 
+    public function getLocationsArray($object, $workspace_id)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/DiagramController.php</br>**Line**: 384</br>Variable "workspace_id" is not in valid camel caps format